README.rst

```diff
@@ -0,0 +1,40 @@
+django-makemessages-xgettext - Generates translation with additional options
+============================================================================
+
+:Authors:
+  Resulto Developpement Web Inc.
+:Version: 0.1
+
+This projects has one goal:
+
+1. Allow makemessages to receive additional parameters to pass to xgettext.
+
+Requirements
+------------
+
+django-makemessages-xgettext works with Python 2.7 and 3.4. It requires Django 1.7+
+
+Installation
+------------
+
+::
+
+    pip install django-makemessages-xgettext
+
+Usage
+-----
+
+As a way to generate .po files without the line number comment reference
```

Comment:
I believe you need to add this app to your INSTALLED_APPS settings in order to work.